pkg/asset/ignition/machine/node.go

```diff
@@ -21,7 +21,7 @@ func pointerIgnitionConfig(installConfig *types.InstallConfig, rootCA []byte, ro
 					Source: func() *url.URL {
 						return &url.URL{
 							Scheme: "https",
-							Host:   fmt.Sprintf("api-int.%s:22623", installConfig.ClusterDomain()),
+							Host:   fmt.Sprintf("%s:22623", installConfig.BareMetal.ApiVIP),
```

Comment:
We need a solution that we can bring to openshift/installer, can we add a generic per-platform configuration which defaults to api-int except for baremetal?

Reply:
Also, just asking, but couldn't we use `api.${CLUSTER_DOMAIN}` instead of the VIP for baremetal? Then no need to add IP's to certs, etc.

Reply:
No, api-int was added because we couldn't use api for internal communication. It had to do with possible user-supplied certificates on the api endpoint that wouldn't be trusted by our nodes. There's more discussion in https://bugzilla.redhat.com/show_bug.cgi?id=1685704